examples/nullable.md

```diff
@@ -0,0 +1,45 @@
+# Nullable type attribute
+
+The `nullable` type attribute can be used to outline the instance of this type *Value* MAY be unset (e.g. `null` or `nil`).
+
+## Unset a `string` or an `array` type
+
+### MSON
+
+```mson
+- keyA (string, nullable, required) - Value of this field can be either unset or of the `string` type
```

Comment:
I'm wondering if it would make sense to reword this slightly.

> Can be either unset or of the `string` type

This states the value can be unset, my interpretation of "unset" is that keyA can be removed from the object such as the following JSON example:

```json
{
  "keyB": "sdfdsf"
}
```

Which isn't what you've intended, I think saying "can either be `null` or of the `string` type might be clearer.